### ISSUE TITLE
[OLD] Legacy Ukrainian Translation Progress

### ISSUE THREAD

**Issue by tesseralis** [reactjs/uk.react.dev#1]
## Maintainer List

* @vldmrkl 

## For New Translators

To translate a page:
1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate.
3. Clone this repo, translate your page, and submit a pull request!
<pre><b>Please take only one page at a time.</b></pre>

Before contributing, read the [glossary and style guide](https://github.com/reactjs/uk.reactjs.org/blob/master/TRANSLATION.md) to understand how to translate various technical and React-specific terms.

Please be prompt with your translations! If you find that you can't commit anymore, let the maintainers know so they can assign the page to someone else.

## Core Pages

To do before releasing as an "official" translation. **Please translate these pages first.**

* [x] Home Page (@oleksii-polovyi) #6
* [x] Tutorial (@marichka-offen) #85 

### Basics

* [x] Hello World  (@ithompson4) #4 
* [x] Introducing JSX (@westanvv ) #3 
* [x] Rendering Elements (@mihilbabin) #2 
* [x] Components and Props (@westanvv) #10 
* [x] State and Lifecycle (@MargarytaChepiga) #75 
* [x] Handling Events (@mihilbabin) #9 
* [x] Conditional Rendering (@vldmrkl) #8 
* [x] Lists and Keys (@ithompson4) #21 
* [x] Forms (@victorpavlov) #112 
* [x] Lifting State Up (@westanvv) #102 
* [x] Composition vs Inheritance (@mihilbabin) #12 
* [x] Thinking in React (@vldmrkl) #24 

### API Reference

* [x] React (@liashchynskyi) #17 
* [x] React.Component (@mihilbabin) #31 
* [x] ReactDOM (@Kiwka) #38 
* [x] ReactDOMServer (@vldmrkl) #54 
* [x] DOM Elements (@victorpavlov) #35 
* [x] SyntheticEvent (@TheTonnio) #36
* [x] Test Utilities (@liashchynskyi) #30 
* [x] Shallow Renderer (@oleksii-polovyi) #26 
* [x] Test Renderer (@victorpavlov) #65 
* [x] JS Environment Requirements (@liashchynskyi) #19 
* [x] Glossary (@mihilbabin) #55 

### Navigation

These are the navigation links that appear in the sidebar. Possibly wait until
the corresponding sections are translated to do these.

* [x] tutorial/nav.yml (@vldmrkl) #99 
* [x] docs/nav.yml (@vldmrkl) #71 

## Next Steps

These are the next most important translations, ordered by importance:

### Installation

* [x] Getting Started (@Kurzdor, @PetRichOst, @vldmrkl) #93 
* [x] Add React to a Website (@Kurzdor) #33 
* [x] Create a New React App (@TheTonnio) #62 
* [x] CDN Links (@ptaberg) #79

### Hooks

Everyone is hooked on hooks! People are really excited about these and it would be great for them to be translated.

  * [x] Introducing Hooks (@UnforbiddenYet) #95
  * [x] Hooks at a Glance (@vldmrkl) #63 
  * [x] Using the State Hook (@mihilbabin) #61 
  * [ ] Using the Effect Hook (@Kurzdor, @oleksii-polovyi) #152
  * [x] Rules of Hooks (@victorpavlov) #72
  * [x] Building Your Own Hooks (@mihilbabin) #76 
  * [x] Hooks API Reference (@mihilbabin) #88 
  * [x] Hooks FAQ (@mihilbabin) #100

### Concurrent Mode
  * [x] Introducing Concurrent Mode (@HLOkeksandr) #190 
  * [ ] Suspense for Data Fetching
  * [ ] Concurrent UI Patterns (@CoffeeSquanderer) #202 
  * [ ] Adopting Concurrent Mode
  * [ ] Concurrent Mode API Reference

### Warnings

These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so **please don't translate the error message itself**.
 
* [x] dont-call-proptypes (@westanvv) #103
* [x] invalid-aria-prop (@moondef) #165 
* [x] invalid-hook-call-warning (@mikhailnetkachev-halolab) #185
* [x] legacy-factories (@vldmrkl ) #169
* [x] refs-must-have-owner (@moondef) #144 
* [x] special-props (@moondef ) #170
* [x] unknown-prop (@mikhailnetkachev-halolab) #183 

### Advanced Guides

* [ ] Accessibility (@victorpavlov) #206 
* [x] Code-Splitting (@vtereshyn) #15 
* [ ] Context (@so-tati)
* [x] Error Boundaries (@vburlai) #105 
* [x] Forwarding Refs (@pasichnyk) #156 
* [x] Fragments (@necheporenko) #77 
* [x] Higher-Order Components (@mikhailnetkachev-halolab) #182 
* [x] Integrating with Other Libraries (@vborshchov) #199 
* [x] JSX In Depth (@rodynenko) #97 
* [x] Optimizing Performance (@mihilbabin) #114 
* [x] Portals (@necheporenko) #82 
* [x] React Without ES6 (@Kurzdor) #48 
* [x] React Without JSX (@Kurzdor) #46 
* [ ] Reconciliation (@klvsov) #306
* [x] Refs and the DOM (@rodynenko) #123 
* [x] Render Props (@vburlai) #106
* [x] Static Type Checking (@vburlai) #128 
* [x] Strict Mode (@vburlai) #121
* [x] Typechecking With PropTypes (@vburlai) #98 
* [x] Uncontrolled Components (@rodynenko) #133 
* [x] Web Components (@vldmrkl) #111 

### FAQ

* [x] APIs and AJAX (@vldmrkl) #109 
* [x] Babel, JSX, and Build Steps (@vldmrkl) #108 
* [x] Passing Functions to Components (@marichka-offen) #57 
* [x] Component State (@marichka-offen) #66
* [x] Styling and CSS (@vtereshyn) #18 
* [x] File Structure (@vldmrkl) #115
* [ ] Versioning Policy
* [x] Virtual DOM and Internals (@soulshockers) #173

### Contributing

Priority: Low

* [ ] How to Contribute (@lytvyn139) #213 
* [ ] Codebase Overview
* [ ] Implementation Notes
* [ ] Design Principles

### Components

Components in `src/components` that have some text in them.

* [x] CodeEditor (@vldmrkl) #13 
* [x] CodeExample (@vldmrkl) #13 
* [x] ErrorDecoder (@vldmrkl) #13 
* [x] LayoutFooter (@vldmrkl) #13 
* [x] LayoutHeader (@vldmrkl) #13 
* [x] MarkdownPage (@vldmrkl) #13 
 
## Additional Translations

These are not the primary translation targets.

* [ ] Blog
* [ ] Community

**Comment by ApolllonDev** [comment]
I would like to translate `Home Page`.

**Comment by ithompson4** [comment]
I would like to translate `Hello World`.

**Comment by westanvv** [comment]
I would like to help in translation. I'll translate `Introducing JSX`

**Comment by mihilbabin** [comment]
I would like to translate `Rendering Elements`.

**Comment by klymenkoo** [comment]
@ApolllonDev @ithompson4 go for it!

**Comment by ithompson4** [comment]
@klymenkoo, I wanted to translate `Hello World` not `Tutorial`. Could you please edit?

**Comment by westanvv** [comment]
I would like to translate `Components and Props` page

**Comment by klymenkoo** [comment]
@westanvv Great questions. 
`Пропси` should work.

**Comment by klymenkoo** [comment]
Repo with Russian translation has [TRANSLATION.md](https://github.com/reactjs/ru.reactjs.org/blob/master/TRANSLATION.md), which contains agreed translations of termins.
It would nice to have something similar in this repo as well.

**Comment by westanvv** [comment]
> Repo with Russian translation has [TRANSLATION.md] ...
Let's create a similar file. It would be very useful

**Comment by ApolllonDev** [comment]
> Let's create a similar file. It would be very useful

#5 Was created. Please review.

**Comment by ovasylenko** [comment]
Hi guys,
@tri-tium, @timba would like to join you.

**Comment by Tri-tium** [comment]
Hi!
I would like to translate `Tutorial`.

**Comment by MargarytaChepiga** [comment]
If _tutorial_ is taken, I can translate State and Lifecycle

**Comment by mihilbabin** [comment]
Going to pick `Handling Events` next.

**Comment by ApolllonDev** [comment]
@mihilbabin go for it!

**Comment by ithompson4** [comment]
Hello, I would like to translate `Lists and Keys`.

**Comment by ApolllonDev** [comment]
@ithompson4 go for it 👍

**Comment by ovasylenko** [comment]
Hi, Would like to translate  `Conditional Rendering`

**Comment by klymenkoo** [comment]
@ovasylenko that would be awesome! feel free to share your progress and ask any questions 👍

**Comment by westanvv** [comment]
I would like to translate `Lifting State Up`

**Comment by klymenkoo** [comment]
@westanvv sounds good.

**Comment by mihilbabin** [comment]
Moving forward to translating my favorite topic `Composition vs Inheritance`

**Comment by liashchynskyi** [comment]
Hello, I'd like to translate **API Reference** -> `React`

**Comment by klymenkoo** [comment]
@liashchynskyi go for it 👍

**Comment by mihilbabin** [comment]
I'm picking `React.Component` next.

**Comment by vtereshyn** [comment]
Planning to pick `faq-styling` (https://github.com/reactjs/uk.reactjs.org/blob/master/content/docs/faq-styling.md)

**Comment by mihilbabin** [comment]
@klymenkoo @vtereshyn I suggest to reopen #15, since there are still some issues left.

**Comment by liashchynskyi** [comment]
Picking [JavaScript Environment Requirements](https://reactjs.org/docs/javascript-environment-requirements.html)

**Comment by klymenkoo** [comment]
@mihilbabin thanks for the review, good catches 👍Opened #20 
CC: @vtereshyn

**Comment by vtereshyn** [comment]
@klymenkoo , will be fixed in few days, I'm a little bit busy now, sorry

**Comment by mihilbabin** [comment]
I suggest changing the translation of `mutate` to `змінювати` instead of `мутувати` in the translation file.

**Comment by ApolllonDev** [comment]
> I suggest changing the translation of `mutate` to `змінювати` instead of `мутувати` in the translation file.

Please make a new PR and we will review it :smiley:

**Comment by ApolllonDev** [comment]
I would like to translate `Shallow Renderer` next.

**Comment by liashchynskyi** [comment]
I'd like to translate `Test Utilities` next

**Comment by ApolllonDev** [comment]
@liashchynskyi sounds good 👍

**Comment by liashchynskyi** [comment]
@ApolllonDev checkbox `JS Environment Requirements` with my name also))

**Comment by Kiwka** [comment]
Picking `ReactDOM`

**Comment by ApolllonDev** [comment]
@Kiwka go for it 👍

**Comment by victorpavlov** [comment]
Hi! I want translate this chapter ‘DOM Elements’?

**Comment by TheTonnio** [comment]
Hi! I would like to help with **SyntheticEvent** chapter translation.

**Comment by ApolllonDev** [comment]
@victorpavlov, @TheTonnio sounds good 😃

**Comment by Kurzdor** [comment]
Hi!
I would like to help with React Without ES6, React Without JSX and Add React to a Website.
Thanks!

**Comment by ptaberg** [comment]
I translated "CDN Links". Excuse me for the late comment

**Comment by klymenkoo** [comment]
@ptaberg no worries! thanks for contribution 🙌

**Comment by klymenkoo** [comment]
@Kurzdor would be great!

**Comment by Kurzdor** [comment]
Next sections to translate for me are: `Getting Started`, `Create a New React App`, `Context`, `Introducing Hooks`, `Fragments`.

**Comment by liashchynskyi** [comment]
@Kurzdor одна секція за раз!) десь я тут таке читав, якщо не помиляюся))

**Comment by Kurzdor** [comment]
@liashchynskyi ви праві, але все ж таки :)

**Comment by klymenkoo** [comment]
@Kurzdor будь-ласка працюйте над одноюс секцією, а по завершенню переходьте на наступну 🙂

**Comment by UnforbiddenYet** [comment]
@klymenkoo Hey, I'd like to get in and translate the [hooks-intro](https://github.com/reactjs/uk.reactjs.org/blob/master/content/docs/hooks-intro.md)

**Comment by klymenkoo** [comment]
@UnforbiddenYet that would be awesome

**Comment by mihilbabin** [comment]
Just noticed [uk.reactjs.org/content/docs/hello-world.md ](https://github.com/reactjs/uk.reactjs.org/blob/master/content/docs/hello-world.md)- contains the wrong string in the first sentence.

**Comment by ApolllonDev** [comment]
> Just noticed that [uk.reactjs.org/content/docs/hello-world.md ](https://github.com/reactjs/uk.reactjs.org/blob/master/content/docs/hello-world.md)- contains the wrong string in the first sentence.

#51

**Comment by Kurzdor** [comment]
May I start translating the `Getting Started` section?

**Comment by ApolllonDev** [comment]
> @klymenkoo @ApolllonDev May I start translating the `Getting Started` section?

Go for it 👍 
We are still reviewing previous PRs.

**Comment by klymenkoo** [comment]
@Kurzdor let's finish all opened PRs 🙌

**Comment by mihilbabin** [comment]
I'm picking `Glossary` next.

**Comment by Kurzdor** [comment]
@klymenkoo Please check `Add React to a Website` since it was already merged.

**Comment by klymenkoo** [comment]
@Kurzdor let's finish your opened PRs first. I'll review them by EOF tomorrow 🙌

**Comment by klymenkoo** [comment]
@Kurzdor also could you please fix #47?

**Comment by TheTonnio** [comment]
I wanna translate `Create a New React App`.

**Comment by klymenkoo** [comment]
@TheTonnio go for it 😀

**Comment by marichka-offen** [comment]
I would like to translate FAQ Passing Functions to Components

**Comment by klymenkoo** [comment]
@marichka-offen sounds good 🙌

**Comment by victorpavlov** [comment]
Hi, guys! Can I take `Test Renderer` from the API reference?

**Comment by marichka-offen** [comment]
Can I start translation of FAQ: APIs and AJAX; Babel, JSX, and Build Steps and Component State? Or you'd rather me waiting for approval of my previous translation?
I am also asking for three of them because they are very small and each will take no longer than 5-10 min to translate, I'm not trying to be greedy (:

**Comment by klymenkoo** [comment]
@marichka-offen, we have a rule of working on 1 section per time. If they're small, it means that is wouldn't take long to review them and merge 🙂

**Comment by klymenkoo** [comment]
@victorpavlov 👍

**Comment by mihilbabin** [comment]
@klymenkoo, tell me more about splitting :laughing: 
![image](https://user-images.githubusercontent.com/11168258/53823048-a0a0db80-3f79-11e9-88ee-1ba8de0d1654.png)

**Comment by marichka-offen** [comment]
@klymenkoo, I agree that it's more safe to assign one section per person and wait for approval. But at the same time it doesn't sound very productive. It is volunteer work, and one can only devote so much time. What I'm trying to say is that I had time this morning and could finish three sections instead of working/waiting on one section per day. I understand the rule though

**Comment by Kurzdor** [comment]
> @Kurzdor also could you please fix #47?

Fixed.

**Comment by mihilbabin** [comment]
Moving towards `Using the State Hook` translation.

**Comment by klymenkoo** [comment]
@ovasylenko, @Tri-tium, @MargarytaChepiga, @westanvv, hi guys 👋 
How is the progress on your translations? If you have any issues/questions please feel free to ask ;)

**Comment by MargarytaChepiga** [comment]
@klymenkoo Sorry, was busy. WIP right now

**Comment by Kurzdor** [comment]
@klymenkoo I would like to take `Using the Effect Hook` page as the next to translate!
WIP

**Comment by klymenkoo** [comment]
@Kurzdor let's finish all opened PRs first :) 1 PR at a time, right? 😀

**Comment by victorpavlov** [comment]
Hey, guys! I'd like translate `Rules of Hooks`.

**Comment by klymenkoo** [comment]
@victorpavlov it'd be great!

**Comment by mihilbabin** [comment]
`Building Your Own Hooks` are on my list for the next few days.

**Comment by klymenkoo** [comment]
@mihilbabin awesome!

**Comment by ModPhoenix** [comment]
Hi guys! I want to help with translation - **Advanced Guides -> Context**.

**Comment by klymenkoo** [comment]
@ModPhoenix hi 👋go for it :)

**Comment by klymenkoo** [comment]
We haven't heard anything from @westanvv, @ovasylenko and @Tri-tium for a while. So, I'm vacating the following sections from the core documentation:
* Tutorial
* Forms
* Lifting State Up

These documents are up for grabs.

**Comment by marichka-offen** [comment]
@klymenkoo can I take tutorial after my previous PR approved?

**Comment by klymenkoo** [comment]
@marichka-offen sounds good 😀

**Comment by victorpavlov** [comment]
Guys, I can help with Forms from the Basics section, when my last PR to be merged.

**Comment by ApolllonDev** [comment]
@victorpavlov sounds like a plan 👍

**Comment by necheporenko** [comment]
I would like to translate "**Fragments**"

**Comment by necheporenko** [comment]
I would like to translate "**Portals**"

**Comment by westanvv** [comment]
Hello to all!  @klymenkoo, I had some kind of vocation, I'm still working with `Lifting State Up`. Will create PR shortly

**Comment by mihilbabin** [comment]
I will pick `Hooks API Reference` next.

**Comment by klymenkoo** [comment]
@mihilbabin , @westanvv got it, sounds good 🙌

**Comment by vburlai** [comment]
Hi, I would like to contribute but starting with something small - `README.md` to see how it goes.
Hope it's okay, my PR is #87

**Comment by russuSasha** [comment]
Hello. I would like to translate "**Higher-Order Components**"

**Comment by klymenkoo** [comment]
@vburlai absolutely! Thank you for your translation, I'll review it soon.

**Comment by klymenkoo** [comment]
@russuSasha sounds good! Let me know if you have any questions ;)

**Comment by rodynenko** [comment]
Hi! I'd like to translate `JSX In Depth`

**Comment by klymenkoo** [comment]
@rodynenko go for it 👍

**Comment by klymenkoo** [comment]
@UnforbiddenYet hey 👋Do you have any progress on `Introducing Hooks`?

**Comment by UnforbiddenYet** [comment]
@klymenkoo hi there! I'm finishing up, just need to proofread the translation first. I'll do it later tomorrow and I hope it's good to submit.
Hope to get the merge fast cause I've been very picky and poetic on my first translation.

Anyway, you got me. Tomorrow EOD.

**Comment by klymenkoo** [comment]
@UnforbiddenYet sounds good 😀

**Comment by mihilbabin** [comment]
@klymenkoo I will wrap up Hooks translation by translating `Hooks FAQ`.

**Comment by UnforbiddenYet** [comment]
@klymenkooplease check out the _Hooks Into_ - https://github.com/reactjs/uk.reactjs.org/pull/95

**Comment by vburlai** [comment]
Hi, I’d like to translate `Typechecking With PropTypes`

**Comment by klymenkoo** [comment]
@vburlai sounds good.

**Comment by klymenkoo** [comment]
@westanvv any updates? It would be great if you can create a PR with whatever you have now, so we can start reviewing it.

**Comment by vburlai** [comment]
@klymenkoo please check out `Typechecking with Proptypes` here #98

**Comment by klymenkoo** [comment]
Hey, @victorpavlov 👋
Do you have any updates on Forms?

**Comment by victorpavlov** [comment]
Hello, @klymenkoo! Got some overload at work, but now it's easier. So will start and finish soon.

**Comment by klymenkoo** [comment]
@victorpavlov sounds good 👍

**Comment by vburlai** [comment]
Hi @klymenkoo 
I'd like to take `Error Boundaries`.

**Comment by klymenkoo** [comment]
@russuSasha what's up? any updates on HOC?

**Comment by rodynenko** [comment]
Hi! I'd like to translate `Refs and the DOM`

**Comment by westanvv** [comment]
> @westanvv any updates? It would be great if you can create a PR with whatever you have now, so we can start reviewing it.

Hello! @klymenkoo, today I'll create PR

**Comment by westanvv** [comment]
Take a look at [#102](https://github.com/reactjs/uk.reactjs.org/pull/102).
Also I'd like to take `dont-call-proptypes` from Warnings

**Comment by vburlai** [comment]
Hi guys,
Please review #105

**Comment by russuSasha** [comment]
> @russuSasha what's up? any updates on HOC?

In progress

**Comment by vburlai** [comment]
Hi guys,
I would like to translate `Render Props`

**Comment by vburlai** [comment]
Hi @klymenkoo, should I continue working on `Render Props` (it's 50% complete now)? It looks like your name has got assigned to it after my message yesterday.

**Comment by mihilbabin** [comment]
Idk what to pick next. We've covered the best parts already :1st_place_medal:. Will take `Optimizing Performance` from `Advanced Guides` section.

**Comment by klymenkoo** [comment]
@vburlai sorry, my bad. I didn't see your message.
Sure, you can continue working on it 👍

**Comment by vburlai** [comment]
Hi guys,
Please take a look at my PR #106

**Comment by ApolllonDev** [comment]
@victorpavlov how is it going with the `Forms`? Please create a PR with what you have at the moment so we can start reviewing 😃

**Comment by victorpavlov** [comment]
@ApolllonDev Sorry for such a big delay, but have a chance to work on this only now. So I will push PR on Monday.

**Comment by victorpavlov** [comment]
@ApolllonDev Did the PR for Forms #112

**Comment by victorpavlov** [comment]
@ApolllonDev If you don't mind, I'd like to translate  `Accessibility` chapter from the `Advanced Guides` section, when my last PR #112 will be merged.

**Comment by vburlai** [comment]
Hi @klymenkoo 
I'd like to translate `Strict Mode` next

**Comment by klymenkoo** [comment]
Hey, @ModPhoenix 👋Any progress on `Context`?

**Comment by vburlai** [comment]
Congratulations on the milestone 🎉 🎉 🎉 !
Core is now 100% complete.
<img width="326" alt="image" src="https://user-images.githubusercontent.com/3315896/56681470-4098fc80-66c1-11e9-8704-5bde0644fda1.png">

**Comment by ModPhoenix** [comment]
I'll do the PR at the weekend

**Comment by vburlai** [comment]
Hi @klymenkoo
I'd like to translate `Static Type Checking` next.

**Comment by klymenkoo** [comment]
@vburlai sounds good!

**Comment by vburlai** [comment]
Hi @ApolllonDev and @mihilbabin 
`Static Type Checking` PR #128 is ready for reviews.

**Comment by rodynenko** [comment]
Hi! I'd like to translate `Uncontrolled Components`

**Comment by klymenkoo** [comment]
Hi @ModPhoenix, @russuSasha, @victorpavlov 👋 Do you have any progress on your documents?

**Comment by victorpavlov** [comment]
@klymenkoo I started, in progress, but not done yet.

**Comment by russuSasha** [comment]
> Hi @ModPhoenix, @russuSasha, @victorpavlov 👋 Do you have any progress on your documents?

Hi. I am very busy. Please, unassigne "Higher-Order Components" from me

**Comment by pasichnyk** [comment]
I'd like to tranlsate `Higher-Order Components`

**Comment by klymenkoo** [comment]
@pasichnyk sounds good

**Comment by pasichnyk** [comment]
@klymenkoo
What to do with the `HOC` abbreviation, should it be translated as`КВП` or leave as is?

**Comment by klymenkoo** [comment]
@pasichnyk, `КВП`

**Comment by soulshockers** [comment]
I would like to translate "FAQ => Virtual DOM and Internals"

**Comment by HLOkeksandr** [comment]
Hi, Would like to translate "Introducing Concurrent Mode"

**Comment by mikhailnetkachev-halolab** [comment]
I would like to translate "Higher-Order Components".
@klymenkoo @ApolllonDev

**Comment by klymenkoo** [comment]
@HLOkeksandr, @mikhailnetkachev-halolab sounds good! Go for it 🙌

**Comment by mikhailnetkachev-halolab** [comment]
So, I've made 'pull request' with my translation and I would like to take another one page.
Can I take 'unknown-prop'?
@klymenkoo @ApolllonDev

**Comment by mikhailnetkachev-halolab** [comment]
Article "Unknown prop" pulled. Can I get "Invalid Hook Call Warning"?
@klymenkoo @ApolllonDev

**Comment by olgatim** [comment]
Hi. I would like to translate "Suspense for Data Fetching". @klymenkoo @ApolllonDev

**Comment by ApolllonDev** [comment]
@olgatim would you be able to finish the second part of [Using the Effect Hook](https://github.com/reactjs/uk.reactjs.org/pull/152)?

**Comment by olgatim** [comment]
@ApolllonDev Yes, for sure I am able to finish this part.

**Comment by mikhailnetkachev-halolab** [comment]
I am waiting for a review of my pull requests. Can I take "Reconciliation" from Advanced Guides section?
@klymenkoo @ApolllonDev

**Comment by vborshchov** [comment]
Hello, I would like to translate  `Integrating with Other Libraries`.
@ApolllonDev

**Comment by ApolllonDev** [comment]
@vborshchov sounds good, go for it!

**Comment by vborshchov** [comment]
@ApolllonDev @vldmrkl 
This is [PR](https://github.com/reactjs/uk.reactjs.org/pull/199) with translation of `Integrating with Other Libraries` article

**Comment by CoffeeSquanderer** [comment]
@ApolllonDev @vldmrkl 
Hey, I would like to translate the "Concurrent UI Patterns" page.

**Comment by ApolllonDev** [comment]
Hi, @olgatim @victorpavlov @ModPhoenix 👋
Do you have any progress with your translations?
Looking forward to hearing from you 😃

**Comment by olgatim** [comment]
@ApolllonDev Hi, translation of the article is in progress, soon there will be a result

**Comment by victorpavlov** [comment]
@ApolllonDev Hello! Sorry for such a big delay, but I'm still in progress and thinking will able to finish it soon.

**Comment by victorpavlov** [comment]
@ApolllonDev @vldmrkl Guys, here is my PR #206 for Accessibility section translation. Please, review. Thank you!

**Comment by lytvyn139** [comment]
> How to Contribute 
I can take this 🙋‍♂️🇺🇲🇺🇦🔁

**Comment by lytvyn139** [comment]
I can take this 🙋‍♂️🔁🇺🇦

> How to Contribute

**Comment by vldmrkl** [comment]
@lytvyn139 go for it👍

**Comment by lytvyn139** [comment]
I'll do the:
> Codebase Overview

**Comment by lytvyn139** [comment]
> Codebase Overview
translated line-to-line 220 each, but I need some help with git @vldmrkl 🙏

**Comment by lytvyn139** [comment]
is somebody working on ?
> Implementation Notes

**Comment by oleksii-polovyi** [comment]
@lytvyn139 go for it 👍 
Please keep an eye on `contributing-section` PR should there be any comments from @vldmrkl

**Comment by lytvyn139** [comment]
can't do it now, posted 19 days ago, sorry

**Comment by HLOkeksandr** [comment]
@vldmrkl Hi, i would like to translate "Advanced Guides-> Context"

**Comment by vldmrkl** [comment]
@HLOkeksandr 👍

**Comment by xom9ikk** [comment]
@vldmrkl, I can take this **FAQ**  -> `Versioning Policy`

**Comment by klvsov** [comment]
@vldmrkl, Hi, I would like to translate "Advanced Guides - Reconciliation"

**Comment by klvsov** [comment]
@vldmrkl, Hi, I would like to translate Concurrent Mode - Adopting Concurrent Mode

**Comment by klvsov** [comment]
@vldmrkl, Hi, I would like to translate Concurrent Mode - Concurrent Mode API Reference

**Comment by klvsov** [comment]
@vldmrkl, page Context booked a long time ago, but there’s no translation for it. May I have it for translation?

**Comment by vldmrkl** [comment]
Hi, @sergii-zhuravel 👋
I have less free time these days, but I still try to do some maintenance.

Just drop a name of the page you want to translate here, and then just make a PR with the translation 🙌

**Comment by so-tati** [comment]
Hi @vldmrkl, I'm ready to translate [Context](https://en.reactjs.org/docs/context.html) and [Reconciliation](https://en.reactjs.org/docs/reconciliation.html) in a week and a half. Kindly, mark these as assigned to me.
Thanks!

**Comment by vldmrkl** [comment]
Hey, @so-tati. 
There's actually a PR for reconciliation translation already (#306). I haven't had much time to review and update the list of assignments in the issue description. Sorry 😞 
But please feel free to take the Context page, it's yours :)

**Comment by Ev1ch** [comment]
Hey there, @vldmrkl. 

I am interested in "Suspense for Data Fetching" from "Concurrent Mode" section. If it is an actual task and it is still free now, I would be glad to take it.

**Comment by SamGreenberg** [comment]
Hej hi, @vldmrkl
I would like to translate "Design Principles" page. Please, let me know if it's still available.

**Comment by fazan-a** [comment]
https://github.com/reactjs/uk.reactjs.org/#переклад — ", будь ласка,"

**Comment by gaearon** [comment]
Hi folks,

Just a heads-up that we've done our best to bring `main` and `legacy` branches up-to-date with upstream. This brings you to the latest visual design and sets up all the redirects. We don't anticipate more changes necessary on `legacy`. For `main`, there's a few things you need to be aware of:

- Keep in mind translating UI is best done closer to the end. We plan to make non-trivial code changes to the repo over the next few months. So it's best to focus on the content rather than on UI components.
- The homepage is untranslated so ideally it would be good if you could do a pass at that. For now, unfortunately it requires messing with the code file.

Thank you so much for your cooperation!

**Comment by gaearon** [comment]
Thanks to everyone who contributed to the old website translation!
Started a new issue for the new website: https://github.com/reactjs/uk.react.dev/issues/457
